src/components/video-embed/__tests__/use-seconds-watched.test.tsx

```diff
@@ -6,7 +6,7 @@
 import { render, screen, fireEvent } from '@testing-library/react'
 import VideoHooksTestComponent from '../helpers/video-hooks-test-component'
 
-describe('use-seconds-watched', () => {
+describe.skip('use-seconds-watched', () => {
```

Comment:
@aaronvanderlip Can you explain on how exactly this test was being flaky?

Reply:
@RubenSandwich Thank you for the push and upholding high standards. I bailed earlier after seeing this portion of the test https://github.com/hashicorp/dev-portal/pull/2435/files#diff-fe9538a4dd1734722b4ce9bbe692bc6f80ad3c46a9460d83c76f4a7dd3a10c2fR48-R51 fail randomly with off by one errors. So running the test you might get [1,2,3,4,6,7,8,9,10,11]. I believe I have addressed the issue but please check my assumptions.